CMakeLists.txt

```
@@ -87,11 +90,20 @@ install(TARGETS sensors
     LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
     RUNTIME DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION}
 )
+install(TARGETS steering
+    ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+    LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+    RUNTIME DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION}
+)
 
 ## Mark cpp header files for installation
 install(DIRECTORY lib/sensors/include/sensors/
```

Actually, I think I got this wrong previously. I think it should instead be:
```suggestion
install(DIRECTORY lib/sensors/include
```